src/legacy/core_plugins/kibana/public/discover/angular/context/query_parameters/actions.js

```diff
@@ -28,8 +28,7 @@ import {
 } from './constants';
 
 
-export function QueryParameterActionsProvider(indexPatterns) {
-  const { filterManager } = npStart.plugins.data.query;
+export function QueryParameterActionsProvider(queryFilter) {
```

Comment:
Why would you take the code in this direction?
filterManager is the new platform entity that replaces `queryFilter`

Reply:
Improved the code, I think there were some issues while implementing the PR, however,now it's fine

Reply:
I need to rollback some of the changes to use our local angular's queryFilter, then the functional tests will be green again, problem is the state syncing with the url doesn't work using the filterManager service of NP, currently needs to be done in the local angular instance